chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -392,9 +396,10 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
             props.controlProps?.widgetInstanceId ?? "");
 
         BroadcastService.getMessageByEventName(endChatEventName).subscribe((msg: ICustomEvent) => {
-            console.log("Receiving end chat event", JSON.stringify(msg.payload));
-            if (msg.payload !== uwid.current) {
+            if (msg?.payload?.runtimeId !== TelemetryManager.InternalTelemetryData.lcwRuntimeId) {
```

Comment:
shouldnt this be equal to?, we want to close the chat if the event contains the same id as the runtime, am I wrong?, also  what happens when msg?.payload?.runtimeId is null or undefined?

Reply:
This is to close tab 2 and 3 when the chat is closed on tab 1. `endChatEventName` is raised after tab1 has closed its chat. The other tabs get this end chat command, and reset it's state without ending the chat again through channelConnector